scripts/misc/download_model.py

```diff
@@ -56,6 +57,9 @@ def parse_args() -> argparse.Namespace:
 
     base_parser = argparse.ArgumentParser(add_help=False)
     base_parser.add_argument('--save-dir', type=str, required=True)
+    base_parser.add_argument('--tokenizer-only',
+                             default=False,
+                             action='store_true')
```

Comment:
```suggestion
    base_parser.add_argument('--tokenizer-only',
                             type=bool,
                             default=False,
                             action='store_true')
```

Reply:
apparently, you cannot specify type with action. I initially had [that](https://github.com/mosaicml/llm-foundry/pull/895/commits/936d5e566d0db411f7f5a93e3e2bffce4194b109#diff-b91dc3e13956a84272e7cb09c99a864c10739a26142341d239ffd53596a887eeR59), but had to remove.